### ISSUE TITLE
"you need your hands free" message is blocked by message cooldown

### ISSUE THREAD

**Issue by esotericist** [CleverRaven/Cataclysm-DDA#38807]
# Describe the bug

when attempting to cast a spell with an object in your hands, you will see one (1) instance of that message before it goes on cooldown. subsequent attempts to cast a spell with something in your hand will "silently" fail, if message cooldown is enabled

# Steps To Reproduce

1. learn a spell
2. wield something
3. cast a spell
4. unwield thing
5. cast a spell
6. wield thing again
7. cast spell
8. ponder at silent failure

# Expected behavior

spellcasting-related messages should generally probably bypass message cooldowns

# Screenshots

![image](https://user-images.githubusercontent.com/1569754/76713887-0a9da600-66e1-11ea-9d1d-b6d84470d890.png)

# Versions and configuration

- OS: Linux
    - OS Version: Distributor ID: Ubuntu; Description: Ubuntu 19.10; Release: 19.10; Codename: eoan; 
- Game Version: 0.D-13090-g566b90174a37 [64-bit]
- Graphics Version: Tiles
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Magiclysm [magiclysm],
    eso_ui_bits [eso_ui_bits],
    No Rail Stations [No_Rail_Stations]
]

# Additional context

see #30723 for the pull request where cooldown bypass was added

**Comment by esotericist** [comment]
Other messages also seem to be subject to this, such as
`You lose your concentration`

**Comment by esotericist** [comment]
so yeah. basically all spellcasting related messages are affected by this
![image](https://user-images.githubusercontent.com/1569754/76730856-5bc98c00-6719-11ea-8edb-923f2b51d896.png)

notice how two of the attacks had their success-and-damage lines suppressed.

_personally_ i'm of the opinion combat-relevant messages should never get the cooldown effect

admittedly, it's possible this affects more than just magiclysm, and I didn't notice previously because I used to have message cooldowns disabled. but that's well out of scope for the near term; i think the most important things right now are making sure 'failure' messages bypass cooldown
